File: Semestrul6/ADM/AndroidApplication/src/ro/andrei15193/bitbucketbrowser/model/Repository.java
package ro.andrei15193.bitbucketbrowser.model;

public class Repository
{
	public Repository(String name, String language, String owner, String description)
	{
		_name = name;
		_language = language;
		_owner = owner;
		_description = description;
	}

	public String getName()
	{
		return _name;
	}
	public String getLanguage()
	{
		return _language;
	}
	public String getOwner()
	{
		return _owner;
	}
	public String getDescription()
	{
		return _description;
	}

	private final String _name;
	private final String _language;
	private final String _owner;
	private final String _description;
}

File: Semestrul6/ADM/WindowsPhoneApplication/BitBucketBrowser/BitBucket/Repository.cs
namespace BitBucketBrowser.BitBucket
{
	public class Repository
	{
		public string scm
		{
			get;
			set;
		}
		public bool has_wiki
		{
			get;
			set;
		}
		public string description
		{
			get;
			set;
		}
		public RepositoryLinks links
		{
			get;
			set;
		}
		public string fork_policy
		{
			get;
			set;
		}
		public string language
		{
			get;
			set;
		}
		public string created_on
		{
			get;
			set;
		}
		public string full_name
		{
			get;
			set;
		}
		public bool has_issues
		{
			get;
			set;
		}
		public Owner owner
		{
			get;
			set;
		}
		public string updated_on
		{
			get;
			set;
		}
		public string size
		{
			get;
			set;
		}
		public bool is_private
		{
			get;
			set;
		}
		public string name
		{
			get;
			set;
		}
	}
}